keps/sig-network/20200518-fqdn-in-hostname.md

```diff
@@ -0,0 +1,802 @@
+---
```

Comment:
Put this in kep.yaml file, according to https://github.com/kubernetes/enhancements/tree/master/keps/NNNN-kep-template

The KEP number is the issue number in kubernetes/enhancements

Reply:
Should I create a directory 1792-KEP and put the md and yaml files in it?

Reply:
Yes. This is the new structure since 1.19

Reply:
Fixed now, thanks